packages/broker/src/config/migration.ts

```diff
@@ -81,9 +81,6 @@ const convertV1ToV2 = (source: any): ConfigFile => {
     if (target.httpServer !== undefined) {
         deleteNullProperties(target.httpServer)
     }
-    if (target.plugins.brubeckMiner !== undefined) {
```

Comment:
This probably should not be deleted.

I'll make a separate ticket about dealing with migration in general from brubeck -> 1.0.